package.json

```diff
@@ -5,10 +5,15 @@
   "devDependencies": {
     "@babel/preset-env": "^7.1.6",
     "lerna": "^3.4.3",
-    "prettier": "^1.15.2"
+    "prettier": "^1.15.3",
+    "prettier-plugin-solidity": "^1.0.0-alpha.12"
   },
   "scripts": {
-    "test": "npx lerna run test"
+    "bootstrap": "npx lerna bootstrap",
+    "ganache:start": "npm run ganache:stop && npx lerna run ganache:start --scope tasit-contracts",
+    "ganache:stop": "npx lerna run ganache:stop --scope tasit-contracts",
+    "truffle:migrate": "npx lerna run compile --scope tasit-contracts && npx lerna run migrate  --scope tasit-contracts",
+    "test": "npm run bootstrap && npm run ganache:start && npm run truffle:migrate && npx lerna run test && npm run ganache:stop"
```

Comment:
This flow makes sense reading it here, but it's worth creating a diagram of it too for community members new to this project. I can do that.

Reply:
Ok, after move ganache to root repo seems to me that will be a bit more clear.

Reply:
I created this issue to track the documentation suggestion. I can take the lead on that. https://github.com/tasitlabs/TasitSDK/issues/64